client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,114 @@
-import React from 'react';
+import React, { Component } from "react";
 import "./Navbar.css";
-import Logo from "../Logo/Logo"
+import Logo from "../Logo/Logo";
+import Burger from "../Burger/Burger";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import NavLink from "../NavLink/NavLink";
+import Icon from "../Icon/Icon";
+import PropTypes from "prop-types";
 
-// Function based React Component
-const Navbar = (props) => {
+// Class Based React Component
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    // Default CSS class to apply to the Component
+    this.state = {
+      condensedMenuActive: false,
+      condensedMenu: "",
+      // pageMark controls the triangle that marks which page the user is on.
+      // Passing 0-3 as a prop into this component controls where it displays.
+      pageMark: ["", "", "", ""],
+      // menuItems are careated in order of the following array.
+      menuItems: ["My Directory", "Community", "Forum", "Book a Trip"],
+      // The link end points for each menu item.
+      menuLinks: ["/", "/", "/", "/"],
+      // Each item in this array is a set for the dropdown on each menu link.
+      // This is an example for one drop menu item: { text: 'Explore People', to: '/' }
+      menuDropDown: [[], [], [], []]
```

Comment:
This data may be static, so we may be able to get away with just having it in `render`

Reply:
Did you miss this one?

Reply:
I guess I didn't see the point of changing it yet since we don't have any of the links to put in but I can remove it.